src/nanoarrow/ipc/decoder_test.cc

```diff
@@ -508,7 +595,7 @@ TEST_P(ArrowTypeParameterizedTestFixture, NanoarrowIpcArrowTypeRoundtrip) {
 #endif
 
 std::string ArrowSchemaMetadataToString(const char* metadata) {
-  struct ArrowMetadataReader reader {};
+  struct ArrowMetadataReader reader;
```

Comment:
Not saying this is right or wrong, but I was under the impression clang-tidy would complain about use of an uninitialized variable here